pb/edge_ctrl_pb/edge_ctrl.proto

```diff
@@ -84,10 +88,13 @@ message ApiSessionHeartbeat {
 message SessionAdded {
     bool isFullState = 1;
     repeated Session sessions = 2;
+    string syncStrategy = 3;
+    string syncState = 4;
 }
 
 message SessionRemoved {
     repeated string tokens = 1;
+    repeated string ids = 2;
```

Comment:
Added this id field to enhance logging and make it easier to look up values in router logs. Prior to this only tokens were output which were hard to look up via the API. These are corresponding arrays and not tuples for backwards compact. Old controller + new router won't output the id information. New controller and older router, the id field will be ignored.